### ISSUE TITLE
Accessibility finding: JAWS - Focus moves to top of page upon closing the modal

### ISSUE THREAD

**Issue by coforma-jamie** [department-of-veterans-affairs/va.gov-team#139076]
### Checklist item

ADE-007	Focus returns to trigger when modal is closed

### Severity level

2, High. Should be fixed in 1-2 sprints post-launch.

### Page(s)/url(s)

https://staging.va.gov/manage-dependents/add-remove-form-21-686c-674/confirmation (as an example)

### Details

All pages: When closing the Feedback modal, focus should return to the Feedback button. But using JAWS on the confirmation page, it moves to the top of the page instead.

[Recording](https://coformaco.zoom.us/clips/share/NcqjyzAgQii28UqP8BOQ9Q)

### Reproduction steps

Log in as user 14. Fill out the form. Turn on JAWS and get to the confirmation page. Tab to the "Feedback" button, press enter. Then press enter on the X button to close the modal.

### Proposed solution or next steps

Focus needs to return to the "Feedback" button for all users. This is happening across the website and I'm not sure if it's been reported yet. We'll need to investigate and report it to the proper channels.

**Comment by rileyorr** [comment]
The team reviewed this issue on April 15th and concluded this is likely a platform level issue since it's happening across the site

**Comment by rileyorr** [comment]
I'm going to unassign myself from this ticket; you can tag me directly or reach my team in #benefits-employee-experience with questions or updates on this work!
